scripts/run-sitespeed-test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# This construction of a branch name is is copied from deploy-multidev.
+# This section does not include the error handling present in that file
+# because this script would not be reach if deploy-multidev.
+# exited with an error on the branch name.
+# @todo find out if there is a way to create the variable globally.
+export normalize_branch="$CIRCLE_BRANCH"
+export normalize_branch="${normalize_branch:0:11}"
+#Remove - to avoid failures
+export normalize_branch="${normalize_branch//[-_]}"
+# @todo, check with Rachel on status of programmatically determining which
+# Pantheon domain is used.
+export url="http://"$normalize_branch"-static-docs.gotpantheon.com/docs"
```

Comment:
@rachelwhitton, let me know if you write a way to derive the URL. Yesterday we discussed doing so by processing the result of `terminus site hostnames list`. `terminus site environments` could also work.